docs/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "libretime-docs",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "clear": "docusaurus clear",
+    "serve": "docusaurus serve",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids"
+  },
+  "dependencies": {
+    "@cmfcmf/docusaurus-search-local": "^0.9.3",
+    "@docusaurus/core": "2.0.0-beta.14",
+    "@docusaurus/preset-classic": "2.0.0-beta.14",
+    "@mdx-js/react": "^1.6.21",
+    "clsx": "^1.1.1",
```

Comment:
I think we should use classnames.

Reply:
We do. Clsx is used by the classic Docusaurus theme.

Reply:
I meant using https://github.com/JedWatson/classnames instead of clsx, the latter seem unmaintained.

Reply:
Since Docusaurus is still under active development, I'd say leave that detail to the devs since it ships with the base theme. There aren't really any third party themes at the moment.